politeiawww/api/cms/v1/api.md

```diff
@@ -1424,6 +1425,48 @@ Reply:
 }
 ```
 
+### `CMS Users`
+
+Returns a list of cms users given optional filters. 
+
+**Route:** `GET /v1/cmsusers`
+
+**Params:**
+
+| Parameter | Type | Description | Required |
+|-----------|------|-------------|----------|
+| domain | int | A query int to match against user's domain. | |
+| contractortype | int | A query string to match user's contractor type. | |
+
+**Results:**
+
+| Parameter | Type | Description |
+|-|-|-|
+| users | array of [CMS Abridged User](#cms-abridged-user) | The list of cms users that match the query. 
```

Comment:
The `#cms-abridged-user` should link to the cms abridged user definition.  

www example: 
https://github.com/decred/politeia/blob/d53f86bcb2c057190be558c9f01eabd29a77bd34/politeiawww/api/www/v1/api.md#abridged-user